lib/graphql/queries.js

```diff
@@ -118,6 +118,7 @@ export const loggedInUserQuery = gql`
         settings
         currency
         isDeletable
+        categories
```

Comment:
That's for the Individual user account, this should similarly be added below in `memberOf` for the other profiles (Organization, etc ...).

Reply:
Good catch! Will add.